docs/source/guide/recommender_system_guide.rst

```diff
@@ -0,0 +1,10 @@
+Recommender Systems Guide
+=====================
+
+The recommender system domain is complex and there are a diverse set of problems, use-cases, datasets, and model architectures. In this guide, we provide a summary of the different options and link resources and examples from NVIDIA Merlin to each of them:
+
+.. toctree::
```

Comment:
My $0.02 is that this page is just another click before getting to the good content.

I think you can start by just adding the `recommender_models.rst` to the `toc.yaml` today.  When you have more content, you can just add the pages to the `toc.yaml` instead of using the `toctree` directive.

Also, if a lot more content emerges, consider creating another `:caption:` section in the `toc.yaml`.  I'm lacking inspiration at the moment for a suggested caption and I'm also thinking that this page title, Recommender Systems Guide, is really the purpose for _any_ of the doc in this repo.  There's a bit to think about.

Reply:
@mikemckiernan thanks for your comment.

Yes, you are correct that recommender_system_guide is just one additional click, so far. The idea is that we need much more documentation pages under Recommender System Guide. Recommender Models is the first one. I have in mind:
- Recommender Systems: 4 Stage Recommenders and Deployment
- Scaling Embedding Tables
- Advanced Methods: Multi-Task Learning, Pre-trained Embeddings, Negative Sampling
- etc